requirements.txt

```diff
@@ -7,7 +7,7 @@ scipy==1.9.0
 tabulate==0.8.10
 matplotlib==3.5.2
 palettable==3.3.0
-spglib==1.16.3
+spglib==2.0.1
```

Comment:
Fix in spglib was introduced between v1.16.5 (latest v1) and v2.0.0. Determined using

```sh
pytest pymatgen/analysis/chemenv/coordination_environments/tests/test_coordination_geometry_finder.py -k test_abstract_geometry
```